receiver/elasticsearchreceiver/metadata.yaml

```
@@ -779,3 +779,12 @@ metrics:
       value_type: int
     attributes: [operation, index_aggregation_type]
     enabled: true
+  elasticsearch.index.documents:
+    description: The number of documents for an index.
+    unit: "{documents}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [document_state, index_aggregation_type]
+    enabled: true
```

```suggestion
    enabled: false
```
Please see explanation [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/14873#discussion_r1006849171).